nixos/modules/installer/tools/tools.nix

```diff
@@ -228,19 +272,12 @@ in
     '';
 
     environment.systemPackages =
-      [ nixos-build-vms
-        nixos-install
-        nixos-rebuild
-        nixos-generate-config
-        nixos-version
-        nixos-enter
-      ] ++ lib.optional (nixos-option != null) nixos-option;
-
-    documentation.man.man-db.skipPackages = [ nixos-version ];
-
-    system.build = {
-      inherit nixos-install nixos-generate-config nixos-option nixos-rebuild nixos-enter;
-    };
+      lib.attrValues (lib.mapAttrs (_: tool: tool.package) installerTools);
```

Comment:
```suggestion
    environment.systemPackages = lib.attrValues (lib.mapAttrs (_: tool: tool.package) installerTools);
```
nit

Reply:
I think the line becomes very long like this, making it pretty annoying to read when using multiple panes in an editor.
It seems that there's nothing about line length in the editorconfig file, so not sure what's the convention here.